task_1/anagram/main.go

```diff
@@ -12,9 +18,20 @@ package main
 */
 
 func isAnagram(test, original string) bool {
-	panic("Implement me!")
+
```

Comment:
why we should add empty line in the beginning of the function?
Please remove it